CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Release 5.0.1
```

Comment:
Maybe it's worth adding the release dates to the changelog?  Releases pre 4.0.0 have them.